templates.go

```diff
@@ -524,13 +524,13 @@ coreos:
       ExecStartPre=/bin/bash -c "while [ ! -f /etc/kubernetes/ssl/etcd/server-ca.pem ]; do echo 'Waiting for /etc/kubernetes/ssl/etcd/server-ca.pem to be written' && sleep 1; done"
       ExecStartPre=/bin/bash -c "while [ ! -f /etc/kubernetes/ssl/etcd/server-crt.pem ]; do echo 'Waiting for /etc/kubernetes/ssl/etcd/server-crt.pem to be written' && sleep 1; done"
       ExecStartPre=/bin/bash -c "while [ ! -f /etc/kubernetes/ssl/etcd/server-key.pem ]; do echo 'Waiting for /etc/kubernetes/ssl/etcd/server-key.pem to be written' && sleep 1; done"
-      ExecStart=/usr/bin/etcd2 --advertise-client-urls=https://127.0.0.1:2379,https://127.0.0.1:2380 \
+      ExecStart=/usr/bin/etcd2 --advertise-client-urls=https://0.0.0.0:2379,https://0.0.0.0:2380 \
                                --data-dir=/etc/kubernetes/data/etcd/ \
-                               --initial-advertise-peer-urls=https://127.0.0.1:2380 \
-                               --listen-client-urls=https://0.0.0.0:2379,https://127.0.0.1:2380 \
+                               --initial-advertise-peer-urls=https://0.0.0.0:2380 \
+                               --listen-client-urls=https://0.0.0.0:2379,https://0.0.0.0:2380 \
```

Comment:
I think we just took it from [here](https://github.com/giantswarm/k8scloudconfig/blob/8a20e5e3895796d91b4f1b7dad9d68e0cab7db04/templates.go#L527), without really knowing the reason behind it.